src/Suave/project.json

```diff
@@ -76,7 +76,8 @@
     },
 
     "frameworks": {
-        "netstandard1.6": {
+        "net45": {},
```

Comment:
Apparently not
![image](https://cloud.githubusercontent.com/assets/570470/20036262/b1cea012-a44f-11e6-93a6-9226095c744a.png)